tests/dummy/app/mixins/code-pretty-print.coffee

```diff
@@ -0,0 +1,8 @@
+`import Ember from 'ember'`
+
+CodePrettyPrintMixin = Ember.Mixin.create
+  didInsertElement: ->
+    @_super()
+    Ember.run.next this, -> prettyPrint()
```

Comment:
I think when I made this comment I didn't know what prettyPrint was. It's probably defined globally by the pretty-print dependency ... looks fine